ui/core/components/suggest_reforges_action.tsx

```diff
@@ -525,24 +509,26 @@ export class ReforgeOptimizer {
 									values: [
 										{ name: 'Select preset', value: 0 },
 										...[...statPresets.keys()].map(key => {
-											const ratingValue = statPresets.get(key)!;
-											const percentOrPointsValue = unitStat.convertRatingToPercent(ratingValue)!;
-											const percentValue = unitStat.equalsStat(Stat.StatMasteryRating) ? (percentOrPointsValue * this.player.getMasteryPerPointModifier()) : percentOrPointsValue;
+											const percentOrPointsValue = statPresets.get(key)!;
+											const percentValue = unitStat.equalsStat(Stat.StatMasteryRating)
+												? percentOrPointsValue * this.player.getMasteryPerPointModifier()
+												: percentOrPointsValue;
```

Comment:
Oh also, if the convention is going to be that the units for `statPresets` values will always be percent, then this conversion should not be done, and it should be assumed that the `percentValue` is directly what you are reading from the map.